docs/schemaform/form-config.md

```diff
@@ -0,0 +1,297 @@
+  urlPrefix: '',
+  
+  // The introduction page component to use. Intro page is skipped if not provided
+  introduction: IntroductionComponent,
+  
+  // The confirmation page component to use after form is successfully submitted
+  confirmation: ConfirmationComponent, 
+  
+  // The prefix for Google Analytics events that are sent for different form actions
+  trackingPrefix: '', 
+  
+  // The title of the form. Displayed on all pages
+  title: '', 
+
+  // The subtitle (e.g. form number) of the form. Displayed on all pages, if there's also a title
+  subTitle: '',
+
+  // Schema definitions to include on all pages. Can be overriden using definitions object in
+  // page schema
+  defaultDefinitions: {}, 
+
+  // Object containing the configuration for each chapter. Each property is the key for a chapter
+  chapters: { 
+    
+    // The title of the chapter
+    title: '', 
+    
+    // Object containing the pages for each chapter. Each property is the key for a
+    // page and should be unique across chapters
+    pages: { 
+      
+      // The url for the page
+      path: 'some-path', 
+      
+      // The title of the page. This will show up only on the review page
+      title: '', 
+      
+      // Any initial data that should be set for the form
+      initialData: {}, 
+      
+      // JSON schema object for the page. Follows the JSON Schema format.
+      schema: {
+        type: 'object',
+        properties: {
+          field1: {
+            type: 'string'
+          },
+          // String/boolean/number/array fields that start with view: will be excluded 
+          // from data sent to server
+          // Objects that start with view: will not be sent, but their children will be merged
+          // into the parent object and will be sent
+          'view:field2': {
+            type: 'string'
+          }
+        }
+      },
+      
+      // Object containing the uiSchema for the page. Follows the format in the react-jsonschema-form
+      // docs, with some vets.gov specific additions. See below.
+      uiSchema: {
+        'ui:title': 'My form',
+        field1: {
+          'ui:title': 'My field'
+        }
+      }
+    }
+  }
+}
+```
+
+The `schema` and `uiSchema` objects should have similar structure. In other words, they should have the same fields organized the same way. The main difference between the structure of the two objects is that the uiSchema object does not have to contain all the fields that the schema object does and it does not need a `properties` object for sub-fields. So given this schema, for example:
+
+```js
+{
+  type: 'object',
+  properties: {
+    field1: {
+      type: 'string'
+    }
+  }
+}
+```
+
+The matching uiSchema would be:
+
+```js
+{
+  'ui:title': 'My form',
+  field1: {
+    'ui:title': 'My field'
+  }
+}
+```
+
+This does not apply to array fields; for those, you still need to specify an `items` object that contains the fields for each row in the array.
```

Comment:
Might want to clarify where this `items` object should be included (`uiSchema`?)